XamlStyler.Mac/XamlStyler.Mac.csproj

```diff
@@ -79,7 +79,7 @@
   <ItemGroup>
     <PackageReference Include="Mono.Addins" Version="1.3.8" />
     <PackageReference Include="MonoDevelop.Addins" Version="0.4.7" />
-    <PackageReference Include="Newtonsoft.Json" Version="10.0.3" />
+    <PackageReference Include="Newtonsoft.Json" Version="12.0.3" />
```

Comment:
Is 12.0.3 required? If possible, we should stay consistent between projects, and we cannot move off of 9.0.1 until we drop support for VS2017 (more details in #243).

Reply:
Unfortunately it's required. Package not compiling if we have lower version (see screenshot):
![image](https://user-images.githubusercontent.com/16039381/72090322-7dbc2200-3316-11ea-8d51-8b55b2983989.png)

I guess Visual Studio or MonoDevelop has a dependency on Newtonsoft.Json. Before update of Visual Studio for Mac I had the same error, but it was asking for version 10.0.0.0, after I updated Visual Studio for Mac and MonoDevelop to v8.4, it started to write about version 12.0.0.0

Reply:
Ah, gotcha. No worries then. The way Visual Studio and Newtonsoft versioning are coupled was an interesting rabbit hole I crawled down to understand some odd errors on Windows, but I think the take away was that we should just use the minimum version that works. It doesn't conflict with anything else in the project, so we should be good.